hugr-core/src/export.rs

```diff
@@ -398,8 +463,20 @@ impl<'a> Context<'a> {
             region_children.push(self.export_node(child));
         }
 
-        // TODO: We can determine the type of the region
-        let signature = self.module.insert_term(model::Term::Wildcard);
+        let signature = {
+            let inputs = self.export_type_row(&input_op.types);
+            let outputs = self.export_type_row(&output_op.types);
+            // TODO: What about the extensions?
```

Comment:
extension inference puts the inferred extensions on the parent node signature?

Reply:
Do they always agree? What about an operation that runs a subgraph somewhere else which has different capabilities? In that case the extensions of the dfg region would be different from the extensions of the parent node.

Reply:
Currently they always agree, the semantics of "elsewhere" for extensions aren't really well defined (yet).

Reply:
This should be addressed now.